website/src/views/Home/Home.test.js

```diff
@@ -1,24 +1,99 @@
 import React from 'react';
-import renderer from 'react-test-renderer';
-import { Home } from '..';
-import { shallow, mount, render, configure} from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
-import { Button, Card, Icon } from 'antd';
+import renderer from 'react-test-renderer'; // for snapshot
+import { Home } from '..'; // get home page
+import {
+  shallow,
+  mount,
+  configure,
+} from 'enzyme'; // get stuffs from enzyme
+import Adapter from 'enzyme-adapter-react-16'; // the adapter
+import {
+  Button,
+  Icon,
+} from 'antd'; // require stuff in UI
 
-configure({ adapter: new Adapter() });
+configure({ adapter: new Adapter() }); // configure
 
-describe('Home Page', () => {
-  it('Should render home page without any errors', () => { // eslint-disable-line no-undef
-    expect(renderer.create(<Home />).toJSON()).toMatchSnapshot();
-  });
-  it('should contains a h1 tag with class name Title with text Rails', () => {
-  expect(shallow(<Home />).contains(<h1 className="title">Rails</h1>)).toBe(true)
- })
- it('should contains a button with classname home-button and link to signup', () => {
-   const component = <Button className="home-button" href="/signup">Get Started<Icon type="right-circle" theme="filled" /></Button>
- expect(shallow(<Home />).contains(component)).toBe(true)
-})
- it('should have a class name home-button', () => {
-  expect(shallow(<Home />).exists('.home-button')).toBe(true)
-})
-})
+describe('Home Page', () => { // start the test
+  console.log(shallow(<Home />).debug()); // check the debug (return the html string)
+  console.log(); // for better looking at console.log
+  it('Should render home page without any errors', () => {
+    expect(renderer.create(<Home />).toJSON()).toMatchSnapshot(); // this is the snapshot test
+  });
+  const wrapper = shallow(<Home />);
+  it('should contains a h1 tag with class name Title with text Rails', () => { // contains method
+    expect(wrapper.contains(<h1 className="title">Rails</h1>)).toBe(true); // check if this piece of code(arugment) in the home page or not.
+  });
+  it('should contains a button with classname home-button and link to signup', () => {
+    const component = (
+      <Button href="/signup">
+      Get Started
+        <Icon type="right-circle" theme="filled" />
+      </Button>
+    );
+    expect(wrapper.contains(component)).toBe(false);
+    // this false because it missing a classname of button.
+  });
+  it('should have a class name home-button', () => { // exists method
+    expect(wrapper.exists('.home-button')).toBe(true); // this checks if this classname in the homepage or not
+  });
+  it('should find the class home', () => { // find method
+    expect(wrapper.find('.home').length).toBe(1); // check if the amount of classnames match to the arugemnt
+  });
+  /* find still have work with id, component, text, object property(props)
+   find can be combine with a lot of other methods */
```

Comment:
For different test case, please leave at place a blank line so it easier to read.

Reply:
Done